tools/wasm-sourcemap.py

```diff
@@ -178,6 +178,19 @@ def remove_dead_entries(entries):
     block_start = cur_entry
 
 
+def parse_debug_info_contents(text):
```

Comment:
since it doesn't parse all the debug info contents (and since "debug info" is a bit ambiguous in this context since it could mean debug info generally or the specific .debug_info section), maybe this function could be called something that indicates that it extracts comp_dir information (e.g. "extract_comp_dir_map" or something like that).

Reply:
Done. Renamed to `extract_comp_dir_map`